llm_bench/python/utils/model_utils.py

```diff
@@ -96,9 +96,6 @@ def set_default_param_for_ov_config(ov_config):
     # With this PR https://github.com/huggingface/optimum-intel/pull/362, we are able to disable model cache
     if 'CACHE_DIR' not in ov_config:
         ov_config['CACHE_DIR'] = ''
-    # OpenVINO self have default value 2 for nstreams on machine with 2 nodes. Reducing memory consumed via changing nstreams to 1.
```

Comment:
can we also drop `PERFORMANCE_HINT` as `LATENCY` is a default for plugin?

Reply:
Do we have a conformance test to verify this LATENCY as default performance hint for plugin?

Reply:
No, we don't check that `LATENCY` is default for `PERFORMANCE_HINT ` now , but we have tests for def values ​​and i will add this property there

Reply:
[PR#25149](https://github.com/openvinotoolkit/openvino/pull/25149)